.github/workflows/generated-windows-binary-wheel-nightly.yml

```diff
@@ -46,7 +46,6 @@ jobs:
       GPU_ARCH_TYPE: cpu
       SKIP_ALL_TESTS: 1
       DESIRED_PYTHON: "3.8"
```

Comment:
This is wrong we need the PYTORCH_EXTRA_INSTALL_REQUIREMENTS here. We want metadata for all the wheels be identical.